### PR TITLE
Proof of concept of the UI

### PR DESCRIPTION
**Pull request by Findeton**
# Changes

* Add linting with `prettier`.
* Add small Readme.
* Add `fontawesome` for fonts and `styled-components` for css.
* Add proof of concept of the UI.

# Screenshots 
Screenshot 2022-06-10 at 09 32 49

### CODE REVIEW

**Comment by edulix** [comment]
Looks good as a proof of concept! Can you please add some instructions on how to reproduce it in a development environment? Probably requires some work related to strand I guess. BTW, minor comment but in the packages.json it's recommended to split `dependencies` and `devDependencies`. This allows to ship less code in the production environment, making it lighter and with less attack surface.

**Comment by Findeton** [comment]
@edulix I already had a `devDependencies`but it looks like the types were added to the other one, good catch. In terms of dev env, I already have some short doc in the readme (but I see it needs a small fix).

File: src/App.test.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+import React from "react"
+import {render, screen} from "@testing-library/react"
+import App from "./App"
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+test("renders learn react link", () => {
```
Comment (edulix) [inline src/App.test.tsx:5]: The test fails when I try it with `yarn test`, maybe it'd be wise to keep it work

**Review by edulix — COMMENTED** [review]
Looks good! Maybe as a good practice we should keep the unit test work even if it's very simple.

**Pull request merged**